### PR TITLE
added new folder

### PR DESCRIPTION
**Pull request by Ajmensah**
## Description
[Please provide a brief description of the changes you've made and the problem you're solving with this pull request.]

## Changes Made
[Please list the specific changes you've made in bullet points or a numbered list.]

## Related Issue
[If your pull request is related to an existing issue, mention it here with a link.]

## Screenshots (if applicable)
[If your changes affect the user interface or visuals, include screenshots to showcase the before and after effects.]

## Testing
[Describe the testing you have performed to ensure the changes work as expected. Include details about the test environment, test cases, and any relevant results.]

## Checklist
Please review and check off the following items before submitting the pull request:
- [x] I have read and followed the project's contributing guidelines.
- [ ] My code follows the project's coding standards and style.
- [ ] I have added necessary documentation or updated existing documentation.
- [x] I have run the appropriate tests and they pass without any failures.
- [ ] My changes do not introduce any new warnings or errors.
- [x] I have rebased my branch on the latest main/master before submitting this pull request.

## Additional Comments (if any)
[Add any additional information or comments you think the reviewers should know about.]

### CODE REVIEW

File: exercise2/anime.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+n the ancient land of Sangoku, where emerald green mountains kissed the heavens and tranquil rivers flowed with wisdom, there lived a humble farmer named Hiroshi. He toiled under the golden sun, tending to his rice fields with unwavering dedication.
+
+One fateful day, as Hiroshi toiled in the fields, he discovered a small, intricately carved stone statue hidden amidst the tall grass. The statue depicted a mythical creature, half-dragon and half-lion, known as a "sangoku." Legend had it that the sangoku possessed extraordinary powers and could bring fortune to those who revered it.
```
Comment (s5carles) [inline exercise2/anime.txt:3]: please add a readme.md file

**Review by s5carles — CHANGES_REQUESTED** [review]

File: exercise2/anime.txt (hunk 1 above)
Comment (Ajmensah) [reply exercise2/anime.txt:3]: Issue resolved

**Review by s5carles — COMMENTED** [review]
good to be merge

**Review by Ajmensah — COMMENTED** [review]
done

**Pull request merged**